src/applications/planning-guidance/8832/config/pages/01-selection.js

```diff
@@ -0,0 +1,21 @@
+export const schema = {
+  type: 'object',
+  properties: {
+    applicantType: {
+      type: 'string',
+      title: 'Select one of the option below',
```

Comment:
This should be options

Reply:
this is all just placeholder code, none of it is permanent and will all change